src/frontend/js/utils/test/factories.ts

```diff
@@ -32,6 +32,9 @@ export const EnrollmentFactory = createSpec({
   created_at: derived(() => faker.date.past()().toISOString()),
   user: faker.datatype.number(),
   course_run: faker.datatype.number(),
+  course: derived(() => CourseFactory.generate()),
+  is_active: true,
+  resource_link: faker.internet.url(),
```

Comment:
You should use `JoanieEnrollmentFactory` instead. Currently we manage two types of enrollment. Those passed by Richie then those passed by Joanie and there are some differences between both so we have two factories ..